pkg/libvmi/network.go

```diff
@@ -139,3 +139,10 @@ func WithAutoAttachPodInterface(enabled bool) Option {
 		vmi.Spec.Domain.Devices.AutoattachPodInterface = &enabled
 	}
 }
+
+// WithIOThreadsPolicy sets the WithIOThreadPolicy parameter
+func WithIOThreadsPolicy(policy kvirtv1.IOThreadsPolicy) Option {
```

Comment:
How is this field [1] related to `network`?

[1] https://github.com/kubevirt/kubevirt/blob/1732a6d12839836ca65241722c3ecfe14df480fd/staging/src/kubevirt.io/api/core/v1/schema.go#L201-L205